examples/xsm.m

```diff
@@ -8,8 +8,7 @@
 	SetDirectory[DirectoryName[$InputFileName]]
 ];
 (*Put this if you want to create multiple model-files with the same kernel*)
```

Comment:
This line should also be removed, shouldn't it?

Reply:
I lege this deliberately on this example. It's just an option.